### PR TITLE
Created associations for classes

### PR DESCRIPTION
**Pull request by gabrielyea**
# Hello!
- Created `Book`, `Rental` and `Classroom` classes
- Created associations between them.
- Update pass classes to handle the reference to its linked class.
- Added sample class `App` just run `ruby app.rb` on the command line!
- Added add_rental method in `Person` and `Book`

### Code
image
### Result
image

### CODE REVIEW

File: book.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```
Comment (DeliceLydia) [inline book.rb]: - [x] Please add rentals method in this class with `person` and `date` as parameters to know who is the person who rent a book and a date he did it

File: person.rb [hunk 2]
```diff
@@ -31,6 +32,3 @@ def validate_name(name)
     @corrector.correct_name(name)
   end
```
Comment (DeliceLydia) [inline person.rb:34]: - [x] Please add rentals method in this class with `book` and `date` as parameters

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
# Hi @gabrielyea, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._


### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

To Highlight: 
- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashboard. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**